api/client/model_alert_condition_metric_type.go

```diff
@@ -13,9 +13,9 @@ type AlertConditionMetricType string
 
 // List of AlertConditionMetricType
 const (
-	ThroughputAlertConditionMetricType AlertConditionMetricType = "throughput"
-	LatencyAlertConditionMetricType    AlertConditionMetricType = "latency"
-	ErrorRateAlertConditionMetricType  AlertConditionMetricType = "error_rate"
-	CPUAlertConditionMetricType        AlertConditionMetricType = "cpu"
-	MemoryAlertConditionMetricType     AlertConditionMetricType = "memory"
+	THROUGHPUT_AlertConditionMetricType AlertConditionMetricType = "throughput"
```

Comment:
Can you exclude modification that changes the constant name? The swagger generated constant names are strangely not following convention.

Reply:
Sorry to trouble you but how do I go about doing this?

Reply:
Just don't submit the changes. @ariefrahmansyah Do you remember which swagger version that produce the correct convention?

Reply:
Swagger produces constants with this format. We have CamelCase previously because we manually fix the linting error: https://github.com/gojek/merlin/commit/dc6f4c3bc3730c67cccdb7e0b106014abf90fd48#diff-bcbc448bedb0c6b16d1c8635ecb2fcd9567044f36d628d6a695ad0477f413361

I think what we can do is:
1. Do not commit these API's client change (We can cherry-pick the change)
2. Entirely ignore API's client directory from linting

I prefer to choose option number 1. I think we still need to check and lint the API's client to make sure the generated code is correct and expected.

Reply:
alright i have changed this.